services/app/src/components/views/quest/cardtemplates/combat/Victory.tsx

```diff
@@ -20,63 +22,103 @@ export interface DispatchProps {
-      contents.push(<p key="c1">All adventurers <strong>regain {props.victoryParameters.heal} health</strong> (even if at 0 health).</p>);
-    } else if (props.victoryParameters.heal === 0) {
-      contents.push(<p key="c1">Adventurers <strong>do not heal</strong>.</p>);
-    } else {
-      contents.push(<p key="c1">All adventurers heal to full health (even if at 0 health).</p>);
-    }
+function maybeRenderPersona(props: Props): JSX.Element|null {
+  if (!props.settings.contentSets.horror) {
+    return null;
+  }
 
-    if (theHorror) {
-      // Mimic <instruction> appearance
-      contents.push(<div key="c1_horror" className="callout"><p><img className="inline_icon" src={`images/horror_white_small.svg`} /></p><div className="text"><p>The Horror: Keep your current Persona level.</p></div></div>);
-    }
+  return (
+    <span>
+      <img className="inline_icon" src={'images/' + formatImg('horror', props.theme) + '.svg'} /><strong>The Horror:</strong> Keep your current Persona level.
+    </span>
+  );
+}
 
-    if (props.victoryParameters.loot !== false && props.combat.loot && props.combat.loot.length > 0) {
-      contents.push(
-        <p key="c4">The party draws the following loot:</p>
-      );
-      const renderedLoot = props.combat.loot.map((loot: Loot, index: number) => {
-        return (<li key={index}><strong>{capitalizeFirstLetter(numberToWord(loot.count))} tier {NUMERALS[loot.tier]} loot</strong></li>);
-      });
-      contents.push(<ul key="c5">{renderedLoot}</ul>);
+function maybeRenderLoot(props: Props): JSX.Element|null {
+  if (!(props.victoryParameters.loot !== false && props.combat.loot && props.combat.loot.length > 0)) {
+    return null;
+  }
 
-      if (props.settings.showHelp) {
-        contents.push(
-          <span key="c6">
-            <p>Divide the loot amongst adventurers. It can be used at any time and does not cost an action (unless otherwise specified).</p>
-          </span>
-        );
-      }
-    }
+  const renderedLoot = props.combat.loot.map((loot: Loot, index: number) => {
+    return (<li key={index}><strong>{capitalizeFirstLetter(numberToWord(loot.count))} tier {NUMERALS[loot.tier]} loot</strong></li>);
+  });
 
-    if (props.victoryParameters.xp !== false && props.combat.levelUp) {
-      contents.push(<span key="c2"><h3>LEVEL UP!</h3><p>All adventurers may learn a new ability:</p></span>);
-      if (props.settings.showHelp) {
-        contents.push(
-          <ul key="c3">
-            <li>Draw 3 abilities from one of the decks listed on your adventurer card.</li>
-            {theHorror && <ul>
-              <li><strong>The Horror:</strong> All adventurers may also draw from the Influence deck.</li>
+  return (
+    <span>
+      <h2>Draw Loot <img className="inline_icon" src={'images/' + formatImg('loot', props.theme) + '.svg'}></img></h2>
+      <p>The party draws the following loot:</p>
```

Comment:
Same thinking, perhaps we could streamline this to **Draw** four tier I loot, two tier II loot, and one tier III loot? A little bit more complex logic for sentence creation - but, would take up noticeably less space (since the above example would currently take up 4 lines, and the new sentence would take up one, maybe two lines)

Reply:
Let's just remove the leader for now.